00_hello_world/main.go

```diff
@@ -9,5 +9,20 @@ import (
 var out io.Writer = os.Stdout
 
 func main() {
-	fmt.Fprintln(out, "Hallo du schöne Welt!")
+
+	fibonacciSeries(7)
+}
+
+func fibonacciSeries(number int) {
+	var firstNumber = 1
+	var secondNumber = 1
+	fmt.Fprintln(out, firstNumber)
+	fmt.Fprintln(out, secondNumber)
```

Comment:
You can fold these into the loop by outputting `firstNumber` instead of `sum` inside the loop. You can then also, more conventionally, start `count` at zero and test `count < number`.

Reply:
I have changed the start count to zero but have kept the output firstNumber at the same location